### ISSUE TITLE
Graphs have insufficient text alternatives [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#198]
The graphs found on the “NY COVID Data” screen have insufficient text alternatives. For example, the “Total persons tested” graph has an accessible name of “594 undefined, Graph showing daily tests over time”. The actual data of the graph is unavailable to VoiceOver users. For example, the accessible name could be “Daily tests over time, August 01 600, August 07 650, August 13 1000, August 19 1200, August 25 950, August 29 1100”. The example accessible name doesn’t convey all the graph data (i.e., only data for 6 days out of the month), but providing data for each day would make the accessible name excessively verbose.

![“Total persons tested” graph announced by VoiceOver as “594 undefined, Graph showing daily tests over time”](https://user-images.githubusercontent.com/4437084/91874721-916f1c80-ec48-11ea-868c-ee1064b56161.png)

Use the accessibilityLabel attribute on the image to give it a proper accessible name.

```javascript
<Image
  source={…}
  accessibilityLabel='Daily tests over time, August 01 600, August 07 650, August 13 1000, August 19 1200, August 25 950, August 29 1100' />
```

Alternatively, provide a separate view where the data is provided in text form. The latter method has the advantage of allowing more data to be available to VoiceOver users (i.e, they could access data for each day in August).

The “Daily Figure” and “7 day average” below each graph can be removed from the reading order, as the information it provides is not useful for users who can’t see the graph. Use the accessibilityElementsHidden attribute to hide elements from VoiceOver users.

```javascript
<View accessibilityElementsHidden={true}>
  <Text>Daily Figure</Text>
<View>
```

**Comment by AlanSl** [comment]
Done in https://github.com/project-vagabond/covid-green-app/pull/205 and should be fixed in next build
